modules/rostests/tests/sertest/sertest.c

```diff
@@ -56,9 +59,12 @@ int main(int argc, char *argv[])
     if (!bResult) {
         printf("WARNING: EscapeCommFunction(SETRTS) failed: %lx\n", (DWORD)bResult);
     }
+#ifdef TRULY_FIX_ROSTESTS_204
+    // FIXME: Want to use Sleep() instead?
```

Comment:
Yes, just use `Sleep(500)` and get rid of the ifdef stuff

Reply:
Replaced.